nodelet/pkg/pf9kube/pf9/pf9-kube/os_centos.sh

```diff
@@ -392,3 +392,14 @@ EOF
     systemctl daemon-reload
     systemctl restart containerd
 }
+
+function install_keepalived()
+{
+  echo "Removing keepalived"
+  # remove keepalived
+  yum erase -y keepalived
+
+  echo "Installing keepalived"
+  # install keepalived
+  yum install -y $KEEPALIVED_PACKAGE_DIR/keepalived-2.1.3-1.el7.x86_64.rpm
```

Comment:
I don't like this approach. Neither here not in the other PR that was merged. Any change in version will need to me made in multiple places. A better approach would be to define a placeholder in `defaults.env`. The placeholder should then get updated by the `Makefile` and the value from `defaults.env` is then used from all shell scripts. That way just changing the `Makefile` is sufficient.